index.html

```diff
@@ -775,7 +775,9 @@ <h2>
           <a>scope</a> is missing, the navigation scope will be
           <code>/pages/</code> on the same origin. If <a>start_url</a> is
           <code>/pages/</code> (the trailing slash is important!), the
-          navigation scope will be <code>/pages/</code>.
+          navigation scope will be <code>/pages/</code>. If both
```

Comment:
Does this note need to mention anything about same-origin with document URL for now?

Reply:
Fixed.